<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CorePageWindow</class>
 <widget class="QMainWindow" name="CorePageWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>320</width>
    <height>800</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>320</width>
    <height>500</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>320</width>
    <height>800</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="minimumSize">
    <size>
     <width>320</width>
     <height>500</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>320</width>
     <height>800</height>
    </size>
   </property>
   <widget class="QFrame" name="content_core_frame">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>300</width>
      <height>750</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>300</width>
      <height>500</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>300</width>
      <height>770</height>
     </size>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QWidget" name="list_top" native="true">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>300</width>
       <height>200</height>
      </rect>
     </property>
    </widget>
    <widget class="QWidget" name="list_content" native="true">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>200</y>
       <width>300</width>
       <height>600</height>
      </rect>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
